SmartDeviceLink/SDLOnDriverDistraction.h

```diff
@@ -28,6 +28,20 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (strong, nonatomic) SDLDriverDistractionState state;
 
+/**
+ If enabled, the lock screen will be able to be dismissed while connected to SDL, allowing users
+ the ability to interact with the app.
```

Comment:
Documentation paragraphs should all be on one line and allowed to automatically wrap.